### ISSUE TITLE
How use with Gulp?

### ISSUE THREAD

**Issue by raulghm** [suitcss/preprocessor#30]
Currently I still using this way:

```
gulp.task('suitcss', ['postcss'], () => {
  suitcss(fs.readFileSync(entryFile.css, 'utf8'), {
    minify: false,
  }).then(function(output) {
    fs.writeFileSync(outputFile.css, output.css);
  });
});
````
Is possible use with the gulp pipeline streaming, can you provide an example?

**Comment by simonsmith** [comment]
Hey,

It's on my list to refactor https://github.com/simonsmith/gulp-suitcss to use the latest `suitcss` preprocessor but PRs welcome in the meantime :)

**Comment by simonsmith** [comment]
Fixed in [2.0.0](https://github.com/simonsmith/gulp-suitcss/releases/tag/2.0.0)
